opentelemetry-journald-logs/src/lib.rs

```diff
@@ -0,0 +1,292 @@
+use async_trait::async_trait;
+use libc::{c_int, c_void};
+use opentelemetry::logs::AnyValue;
+use std::ffi::CString;
+use std::fmt::Debug;
+
+use opentelemetry::logs::LogError;
+use opentelemetry_sdk::export::logs::{LogData, LogExporter};
+
+use opentelemetry::logs::Severity;
+
+#[cfg(feature = "json")]
+use serde::Serialize;
+
+extern "C" {
+    fn sd_journal_sendv(iov: *const libc::iovec, n: libc::c_int) -> libc::c_int;
+}
+
+#[derive(Default)]
+pub struct JournaldLogExporterBuilder {
+    identifier: Option<String>,
+    message_size_limit: Option<usize>,
+    attribute_prefix: Option<String>,
+    json_format: bool,
+}
+
+impl JournaldLogExporterBuilder {
+    pub fn identifier(mut self, identifier: &str) -> Self {
+        self.identifier = Some(identifier.to_string());
+        self
+    }
+
+    pub fn message_size_limit(mut self, message_size_limit: usize) -> Self {
+        self.message_size_limit = Some(message_size_limit);
+        self
+    }
+
+    pub fn attribute_prefix(mut self, attribute_prefix: Option<String>) -> Self {
+        if let Some(prefix) = attribute_prefix {
+            self.attribute_prefix = Some(prefix.to_uppercase());
+        } else {
+            self.attribute_prefix = None;
+        }
+        self
+    }
+
+    pub fn json_format(mut self, json_format: bool) -> Self {
+        self.json_format = json_format;
+        self
+    }
+
+    pub fn build(self) -> Result<JournaldLogExporter, &'static str> {
```

Comment:
Could we simply return `JournadLogExporter` instead of a `Result`?

```suggestion
    pub fn build(self) -> JournaldLogExporter {
```

Reply:
Done